### PR TITLE
Remove Backdoor user PBA

### PR DESCRIPTION
**Pull request by ilija-lazoroski**
**Merge #1434 before this to avoid messy merge conflicts!**

---

# What does this PR do? 

Fixes part of #1410 . 

Add any further explanations here. 

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing? 
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [x] Was the documentation framework updated to reflect the changes?

## Testing Checklist

* [x] Added relevant unit tests?
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by {Running the Monkey locally with relevant config/running Island/...} 
* [x] If applicable, add screenshots or log transcripts of the feature working

## Explain Changes

image
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -67,10 +67,12 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - Improve runtime of some unit tests. #1125
 - Run curl OR wget (not both) when attempting to communicate as a new user on
   Linux. #1407
+- Renamed Communicate as new user to Communicate as backdoor user. #1410
```
Comment (shreyamalviya) [inline CHANGELOG.md:70]: In the CHANGELOG, we track the PR number, not the issue number. :)

```suggestion
- Renamed CommunicateAsNewUser to CommunicateAsBackdoorUser. #1433
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -67,10 +67,12 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
...
 
 ### Removed
 - Relevant dead code as reported by Vulture. #1149
 - Island logger config and --logger-config CLI option. #1151
+- Backdoor user post breach action. #1410
```
Comment (shreyamalviya) [inline CHANGELOG.md:75]:
```suggestion
- Backdoor user post breach action. #1433
```

File: docs/content/development/adding-post-breach-actions.md [hunk 3]
```diff
@@ -39,9 +39,9 @@ class MyNewPba(PBA):
 
 #### Implementation
 
-If your PBA consists only of simple shell commands, you can reuse the generic PBA by passing the commands into the constructor. See the `add_user.py` PBA for reference.
+If your PBA consists only of simple shell commands, you can reuse the generic PBA by passing the commands into the constructor. See the `timestomping.py` PBA for reference.
```
Comment (shreyamalviya) [inline docs/content/development/adding-post-breach-actions.md:42]: Timestomping uses scripts which is slightly more complicated.

Job scheduling and account discovery, (1) exist for both Windows and Linux, and (2) are the easiest ones. Let's use one of those as an example here. You choose. :)

**Review by shreyamalviya — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mssalvatore) [reply CHANGELOG.md:70]: @shreyamalviya Incorrect. Track the issue number when possible. The issue links to the PR and generally gives more context about why the change was made.

This is a decision made on a case-by-case basis, however. The goal is to communicate information about the change to the user.

Comment (shreyamalviya) [reply CHANGELOG.md]: Oh, the impression I gathered from the CHANGELOG was that we always link the PR number. My bad!

Comment (mssalvatore) [reply CHANGELOG.md]: It's on a case-by-case basis. We should prefer the issue number. However, we should use whatever provides the clearest information/story to the reader.

**Review by mssalvatore — APPROVED** [review]

**Pull request merged**